assets/emojis/index.ts

```diff
@@ -26,10 +29,10 @@ const emojiCodeTableWithSkinTones = emojis.reduce<EmojiTable>((prev, cur) => {
     return newValue;
 }, {});
 
-const localeEmojis = {
+const localeEmojis: LocaleEmojis = {
```

Comment:
Why was this needed?

Reply:
Otherwise, I had a TS error in [this function](https://github.com/VickyStash/Expensify/blob/ts-migration/reactions-component/src/libs/EmojiUtils.ts#L57). I guess it also could be resolved with assertion/additional conditions if you think type update isn't a good idea

Reply:
It's fine as you coded 👍